oak_functions/examples/key_value_lookup/module/src/tests.rs

```diff
@@ -70,6 +70,7 @@ async fn test_server() {
     let server_background = test_utils::background(|term| async move {
         create_and_start_grpc_server(
             &address,
+            vec![],
```

Comment:
Could you add a comment, what this argument refers to?

Reply:
Declared a parameter instead. BTW, is tee_certificate that is used for creating `AttestationServer` ever used?